### ISSUE TITLE
[Mobile] [Bug] App Becomes Stuck On Pressing Accessibility Button

### ISSUE THREAD

**Issue by NLUG18HV9X7629CA6SP8TN3Y** [standardnotes/forum#1753]
On Android 12 when I am in the app and press the accessibility button for color correction (an accessibility button you can turn on in the settings on Android) the app goes to a blank screen and you are unable to do anything further in the app without restarting it.

**Comment by myreli** [comment]
Hi @NLUG18HV9X7629CA6SP8TN3Y! It seems like this issue no longer happens with the current app version, probably fixed with the new mobile experience. 

Can you confirm if it works as expected now?
